src/webui/www/private/confirmtrackerdeletion.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8">
+    <title>QBT_TR(Remove tracker)QBT_TR[CONTEXT=confirmDeletionDlg]</title>
+    <link rel="stylesheet" href="css/style.css?v=${CACHEID}" type="text/css">
+    <script src="scripts/lib/MooTools-Core-1.6.0-compat-compressed.js"></script>
+    <script src="scripts/lib/MooTools-More-1.6.0-compat-compressed.js"></script>
+    <script>
+        "use strict";
+
+        window.addEventListener("DOMContentLoaded", () => {
+            const url = new URI().getData("url");
+            $("confirmDeleteTrackerText").textContent = 'QBT_TR(Are you sure you want to remove tracker %1 from all torrents?)QBT_TR[CONTEXT=TrackersFilterWidget]'.replace("%1", `"${window.parent.qBittorrent.Misc.escapeHtml(decodeURIComponent(url))}"`);
```

Comment:
With `textContent` it doesn't need escaping anymore:
```suggestion
            $("confirmDeleteTrackerText").textContent = 'QBT_TR(Are you sure you want to remove tracker %1 from all torrents?)QBT_TR[CONTEXT=TrackersFilterWidget]'.replace("%1", `"${decodeURIComponent(url)}"`);
```

Reply:
Forgot to remove this.